app/Http/Controllers/CalendarsController.php

```diff
@@ -89,24 +87,24 @@ public function edit($id)
     /**
      * Update the specified resource in storage.
      *
-     * @param  \Illuminate\Http\Request  $request
+     * @param  UpdateCalendarRequest     $request
      * @param  int                       $id
      * @return \Illuminate\Http\Response
      */
     public function update(UpdateCalendarRequest $request, $id)
     {
         $input = $request->input();
 
-        $success = $this->calendars->update($id, $input);
-
         // If events are passed, bulk upsert them
```

Comment:
"insert" instead of "upsert"?

Reply:
Comment comes from legacy code, but we might as well change it.